client/voter/README.md

```diff
@@ -0,0 +1,301 @@
+## Tangle Voter CLI
+
+A simple CLI tool for voting for changing the current Signature Bridge governor.
+
+
+### Build
+
+```
+cargo build --release -p tangle-voter
+```
+
+### How to use the CLI
+
+1. Create a vote by inspecting the refresh proposal:
```

Comment:
If we have a multi-step flow or multiple CLI commands, I would make them more pronounced and also list them all at the top.

Reply:
Done.